dds/DCPS/XTypes/TypeObject.h

```diff
@@ -868,7 +868,7 @@ namespace XTypes {
     ACE_CDR::Boolean boolean_value;
     ACE_CDR::Octet byte_value;
     ACE_CDR::Short int16_value;
-    ACE_CDR::UShort uint_16_value;
+    ACE_CDR::UShort uint16_value;
```

Comment:
This is from the spec

Reply:
We decided to do this change and filed a spec issue.